util/logging.go

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2018 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+// Package util provides ancillary functionality to go-vcloud-director library
+// logging.go regulates logging for the whole library.
+// See LOGGING.md for detailed usage
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"runtime"
+	"strings"
+	"time"
+)
+
+const (
+	// Name of the environment variable that enables logging
+	envUseLog = "GOVCD_LOG"
+
+	// Name of the environment variable with the log file name
+	envLogFileName = "GOVCD_LOG_FILE"
+
+	// Name of the environment variable with the screen output
+	envLogOnScreen = "GOVCD_LOG_ON_SCREEN"
+
+	// Name of the environment variable that enables logging of passwords
+	envLogPasswords = "GOVCD_LOG_PASSWORDS"
```

Comment:
Do you set GOVCD_LOG_PASSWORDS to true ?

Reply:
no. 
This is the name of the environment variable to change the default, which is
```LogPasswords bool = false```